src/ReadableAttributes.php

```diff
@@ -4,6 +4,9 @@
 
 trait ReadableAttributes
 {
+    /** @var Container */
+    private $attributes;
```

Comment:
I prefer protected 
strictly speaking, this should be public to be compatible but I believe no one uses this property directly :stuck_out_tongue: